src/golang/cmd/server/handler/v2/node_artifact_result_content_get.go

```diff
@@ -160,54 +154,8 @@ func (h *NodeArtifactResultContentGetHandler) Perform(ctx context.Context, inter
 			return emptyResp, http.StatusOK, nil
 		}
 
-		return emptyResp, http.StatusInternalServerError, errors.Wrap(err, "Failed to retrieve data for the artifact result.")
-	} else if !errors.Is(err, storage.ErrObjectDoesNotExist()) {
 		return emptyResp, http.StatusInternalServerError, errors.Wrap(err, "Failed to retrieve data for the artifact result.")
 	}
 
 	return &nodeResultGetResponse{IsDownsampled: isDownsampled, Content: data}, http.StatusOK, nil
 }
-
-// This custom implementation of SendResponse constructs a multipart form response with two fields:
-// 1: "metadata" contains a json serialized blob of artifact result metadata.
-// 2: "data" contains the artifact result data blob generated the serialization method
-// specified in the metadata field.
-func (*NodeArtifactResultContentGetHandler) SendResponse(w http.ResponseWriter, interfaceResp interface{}) {
```

Comment:
Why is this removed?

Reply:
We save  the data in memory, so there's no need to send the content as a form data. Json is sufficient to handle